src/client/react/components/CustomFunctionsDashboard/Summary/DetailsItem.tsx

```diff
@@ -50,22 +52,24 @@ export default class DetailsItem extends Component<IProps, IState> {
       margin-left: ${props.indent};
       word-break: ${props.statusIcon && 'break-all'};
       font-variant: ${props.statusIcon && 'small-caps'};
```

Comment:
Weren't we going to have it be a prop, something like "isTitle"?  rather than relying on status icon?